### PR TITLE
Supports devcontainer.

### PR DESCRIPTION
**Pull request by springcomp**
@innovate-invent  I finally got around to create my devcontainer-based environment.
This allows anyone to start cloning the repository and apply the guidelines from the README to quickly have an up and running development environment.

That also helps me who mostly uses Windows for most tasks.

I have been able to spend a short amount of time navigating around the website.
The continuous scrolling virtual pages is something to get used to, I must say 😉.

### CODE REVIEW

File: .devcontainer/start.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## This devcontainer is mounting the workspace
+## in the /src folder which gets its original contents ignored.
+## We need to recreate the symlink to /usr/local/bin/jp
+
+if [[ ! -f "/src/jp" ]]; then
+  ln -s /usr/local/bin/jp /src/jp
+fi
+
+## The static website pulls most of its contents
+## from the jmespath.spec site and corresponding wiki.
```
Comment (innovate-invent) [inline .devcontainer/start.sh]: might want to 's/site/repo/'

**Review by innovate-invent — APPROVED** [review]
This is great, you even have ncat going.

File: .devcontainer/start.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+fi
+if [[ ! -d "/src/content/wiki" ]]; then
+  git clone --depth 1 https://github.com/jmespath-community/jmespath.spec.wiki.git /src/content/wiki/
+fi
+
+## This repository uses symlinks
+## If you are starting this devcontainer from Windows
+## we fix this transparently here
+
+rm /src/data/functions
```
Comment (jdevillard) [inline .devcontainer/start.sh:26]: beware this might show as a change in Git. 
Consider using Git --assume-unchanged.

**Review by jdevillard — APPROVED** [review]

File: .devcontainer/start.sh (hunk 2 above)
Comment (springcomp) [reply .devcontainer/start.sh:26]: Indeed, was not very "transparent" was it? 😁

**Pull request merged**